helm/observability-bundle/values.yaml

```diff
@@ -65,6 +65,26 @@ apps:
         name: "{{ $.Values.clusterID }}-logging-config"
         namespace: "{{ $.Release.Namespace }}"
 
+  alloyMetrics:
+    appName: alloy-metrics
+    chartName: alloy
+    catalog: giantswarm
+    enabled: false
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/alloy-app
+    version: 0.3.1
+    # a list of extraConfigs for the App,
+    # It can be secret or configmap
+    # https://github.com/giantswarm/rfc/tree/main/multi-layer-app-config#example
+    extraConfigs:
+      - kind: secret
+        name: "{{ $.Values.clusterID }}-logging-secret"
```

Comment:
Secret and configmap names are wrong here

Reply:
Renamed them to `$clusterID-monitoring`

Reply:
Why not follow the other schème clusterID-monitoring-config|secret?